app/Http/Controllers/AuthController.php

```diff
@@ -33,7 +33,7 @@ class AuthController extends BaseController
      *    ),
      * ),
      * @OA\Response(
-     *    response=201,
+     *    response=200,
```

Comment:
Can You explain, why?

Reply:
Well, I forgot that we return HTTP 201 after user was successfully created

Reply:
that is my mistake